discord/ui/select.py

```diff
@@ -86,26 +101,37 @@ class Select(Item[V]):
     """
 
     __item_repr_attributes__: tuple[str, ...] = (
+        "select_type",
         "placeholder",
         "min_values",
         "max_values",
         "options",
+        "channel_types",
         "disabled",
     )
 
     def __init__(
         self,
+        select_type: ComponentType,
```

Comment:
Shouldn't this have a default value to prevent a breaking change?
```suggestion
        select_type: ComponentType = ComponentType.string_select,
```